src/ds323x/alarms.rs

```diff
@@ -178,15 +239,23 @@ where
         when: WeekdayAlarm2,
         matching: Alarm2Matching,
     ) -> Result<(), Error<CommE, PinE>> {
-        if when.weekday < 1 || when.weekday > 7 || when.minute > 59 {
+        let weekday_invalid = when.weekday < 1 || when.weekday > 7;
```

Comment:
Same as above: Could you document here how the values of invalid inputs are adapted when these are ignored by the selected matching strategy?